recipe/meta.yaml

```diff
@@ -12,9 +12,8 @@ source:
   sha256: 80e01b4791513c27eded8517af847530dfdf04c43d99ff132ed9c3085933b75b
 
 build:
-  number: 0
+  number: 1
   # There is no win build for librdkafka at this time.
```

Comment:
after the CI finishes can you please remove this comment? I think that is not valid anymore, we already have that dependency on windows

Reply:
Can you remove the empty ``commands`` on line 35 as well please?